packages/etherscan/src/lib/etherscan.ts

```diff
@@ -62,22 +76,53 @@ function initEtherscan(apiKey: string, baseUrl: string) {
 function formatDate(date: Date) {
   return date.toISOString().split('T')[0];
 }
+/** Transform a YYYY-MM-DD string into a Date */
+function toDate(date: string) {
+  const unixDate = Date.parse(date);
+  return new Date(unixDate);
+}
+/** Transform an Unix timeStamp into a Date */
+function unixToDate(timeStamp: string) {
+  const sec = parseInt(timeStamp);
+  return new Date(sec * 1000);
+}
+/** Transform a string into a bigNumber */
+function toBigNumber(value: string): BigNumber {
+  return BigNumber.from(value);
+}
+/** Transform a string into boolean */
+function toBoolean(bool: '0' | '1'): boolean {
+  return bool === '0' ? false : true;
+}
+/** Transform a string into a number */
+function toNumber(value: string): number {
+  return parseInt(value);
+}
+/** Transform a hexString into a number */
+function hexToNumber(value: string): number {
+  return parseInt(value, 16)
+}
 
 //////////////
 // ACCOUNTS //
 //////////////
 
-function balance(call: Etherscan, address: string, tag: Tag) {
-  return call<string>({ module: 'account', action: 'balance', address, tag });
+async function balance(call: Etherscan, address: string, tag: Tag) {
+  const res = await call<string>({ module: 'account', action: 'balance', address, tag });
+  return toBigNumber(res);
 }
 
-function balanceMulti(call: Etherscan, addresses: string[], tag: Tag) {
+async function balanceMulti(call: Etherscan, addresses: string[], tag: Tag) {
   const address = addresses.join(',');
-  return call<BalanceMultiResponse[]>({ module: 'account', action: 'balancemulti', address, tag });
+  const res = await call<BalanceMultiResponse[]>({ module: 'account', action: 'balancemulti', address, tag });
+  return res.map(data => ({
+    account: data.account,
+    balance: toBigNumber(data.balance),
+  }))
 }
 
 function txList(call: Etherscan, address: string, params: Optional<TxListRequest> = {}) {
-  return call<TransactionResponse[]>({ module: 'account', action: 'txlist', address, ...params });
+  return call<TxListResponse[]>({ module: 'account', action: 'txlist', address, ...params });
```

Comment:
There are no string to transform into number, Bignumber, Date or boolean in the returned value ?